content/en/docs/contributing/localization.md

```diff
@@ -327,42 +354,57 @@ Once you are ready:
    - [ ] Read through
          [Localization](https://opentelemetry.io/docs/contributing/localization/)
          and all other pages in the Contributing section
-   - [ ] Localize site homepage to YOUR_LANGUAGE_NAME
+   - [ ] Localize site homepage (only) to YOUR_LANGUAGE_NAME and submit a PR.
+         For details, see
+         [Localize the homepage](https://opentelemetry.io/docs/contributing/localization/#homepage).
    - [ ] OTel maintainers:
      - [ ] Update `hugo.yaml`
      - [ ] Configure cSpell and other tooling support
      - [ ] Create an issue label for `lang:LANG_ID`
      - [ ] Create org-level group for `LANG_ID` approvers
      - [ ] Update components owners for `content/LANG_ID`
+   - Next step: Localize the
+     [Glossary](https://opentelemetry.io/docs/concepts/glossary/)
```

Comment:
```suggestion
- [ ] Next step: Localize the
     [Glossary](https://opentelemetry.io/docs/concepts/glossary/)
```

Reply:
I found a middle ground: kept the step here, but the step is to open an issue to track the localization of the glossary.